tuna/fin_class.py

```diff
@@ -242,7 +262,13 @@ def create_dumplist(self):
 
     if "applicability" in self.fin_steps:
       self.logger.info("Creating dumplist for: %s", self.fin_steps[0])
-      if not self.set_all_configs():
+      idx = 0
+      num_blk = 1
+      if self.multiproc:
+        idx = self.gpu_id
+        num_blk = self.num_procs.value
+
+      if not self.set_all_configs(idx, num_blk):
         return False
       assert self.all_configs is not None
```

Comment:
Possible to remove this assert and return false? I think we once decided we dont want asserts

Reply:
I'll remove